randovania/games/am2r/logic_database/Industrial Complex.txt

```diff
@@ -1108,7 +1108,24 @@ Extra - minimap_data: [{'x': 67, 'y': 29}, {'x': 67, 'y': 30}, {'x': 68, 'y': 28
   * Layers: default
   * 4-Tiles High Dock to Lower Factory Lava Lake Access/Horizontal Dock to Lower Factory Intersection
   > Horizontal Dock to Lower Factory Hub
-      Hijump With Power Grip Wall
+      Any of the following:
+          Space Jump
+          All of the following:
+              Hi-Jump Boots
+              Power Grip or Walljump (Beginner)
+          # IBJ
+          Infinite Bomb Jumping (Intermediate) and Can Use Bombs
+          # CBJ
+          Charged Bomb Jump (Intermediate) and Can Use Charged Bomb Jump
+          All of the following:
+              Can Use Spider Ball
+              Any of the following:
+                  Bombs or Spring Ball or Mid-Air Morph (Beginner)
+                  Power Bombs and Disabled Entrance Rando
+          # Damage Boost off of Shirk
+          Damage Boost (Expert) and Walljump (Advanced) and Normal Damage ≥ 50
+          # Shinespark
+          Speed Booster and Shinesparking Tricks (Beginner) and Short Charge (Expert)
```

Comment:
just noticed, shouldnt this have a "short charge OR room rando"? afaik you can just charge from the right room  and kiill both the autoad and the shirk with the speedboost/shinespark

Reply:
there is not enough room, you will just run into the autoad unless we also want to add weapon requirements on top of `not entrance rando`

Reply:
would rather not. but couldve sworn that it was possible...